### PR TITLE
Add props to custom className to Field Component

### PR DESCRIPTION
**Pull request by williamcosta**
### Description

In doing the implementation of Jason I saw that we were always rendering the class that forced the fields to be 100% of the width and in case we wanted that a field was not 100% of the width, to avoid overwrite of css in modifier class we created the props `wrapperClassName` to pass custom className.

### Changelog

* Add `wrapperClassName` props to field render custom className

### Printscreen

Before without `wrapperClassName` props:

screen shot 2018-05-25 at 10 19 15

After with `wrapperClassName` props:

screen shot 2018-05-25 at 10 18 43

### CODE REVIEW

File: spec/components/Field.spec.js [hunk 1]
```diff
@@ -53,5 +53,29 @@ describe('Field', () => {
...
+          <Field label={'test'} id={'bora-pra-action'} isFluid={true}>
+            <Input id={'bora-pra-action'} name={'nameTest'} onFieldChange={() => { }} />
+          </Field>,
+        );
+
+        expect(component.instance().wrapperStyle).toMatch(/form__field--fluid/);
+      });
+    });
+
+    xdescribe('without isFluid', () => {
```
Comment (iondrimba) [inline spec/components/Field.spec.js]: This spec is commented 🦅 👁

[the remark this answers is not in the document]
Comment (williamcosta) [reply spec/components/Field.spec.js]: 😬 sorry!

**Comment by williamcosta** [comment]
@eduardojmatos yes, I agree, we can leave this improvement to fiddle with the style of the form for an upcoming version.

**Comment by eduardojmatos** [comment]
@williamcosta I think that we can do it now... All this code could be avoided if we just expose the className property. 

I really dont agree with this dozen of properties to modify classes, because are rules of UI, dont have any connection with the functionallity of the component.

**Comment by iondrimba** [comment]
@eduardojmatos Can you specify what are those `dozen` properties that modify classes?

**Comment by williamcosta** [comment]
@eduardojmatos I changed the approach to solution a custom className.

**Comment by eduardojmatos** [comment]
@iondrimba it was a figure of speech :) 

We have some behaviors that change `className`, as in Breadcrumb with `handleActiveStyle`, that makes sense to me. What I don't see a good motivation is when we talk about UI that is not related to the business of Jason.

File: src/components/Step.js [hunk 2]
```diff
@@ -36,7 +36,8 @@ export default class Step extends Component {
               key={`field-${index}`}
               label={item.title}
               id={item.id}
-              errorMessage={item.errorMessage}>
+              errorMessage={item.errorMessage}
+              wrapperStyle={item.wrapperStyle} >
```
Comment (eduardojmatos) [inline src/components/Step.js]: `wrapperStyle` can be confused to API. Is it a style inline? Is it a class?

Comment (williamcosta) [reply src/components/Step.js]: @eduardojmatos I changed name props. 👍

**Pull request merged**